scripts/pnpm-context.mjs

```diff
@@ -169,7 +169,7 @@ async function getMetafilesFromPnpmSelector(selector, cwd, options = {}) {
 async function getPackagePathsFromPnpmSelector(selector, cwd) {
   const projects = await readProjects(cwd, [parsePackageSelector(selector, cwd)]);
 
-  return Object.keys(projects.selectedProjectsGraph).map((p) => relative(cwd, p).replaceAll('\\', '/'));
+  return Object.keys(projects.selectedProjectsGraph).map((p) => relative(cwd, p).replace(/\\/g, '/'));
```

Comment:
In order to resolve "replaceAll is not a function".

https://www.designcise.com/web/tutorial/how-to-fix-replaceall-is-not-a-function-javascript-error